alert-engine/src/main/scala/io/phdata/pulse/alertengine/AlertEngineMain.scala

```diff
@@ -25,26 +25,23 @@ import io.phdata.pulse.alertengine.notification.{
   NotificationServices,
   SlackNotificationService
 }
-import org.apache.solr.client.solrj.impl.{
-  CloudSolrServer,
-  HttpClientUtil,
-  Krb5HttpClientConfigurer
-}
+import io.phdata.pulse.common.SolrService
+import io.phdata.pulse.solr.SolrProvider
 
 import scala.io.Source
-import scala.util.Try
 
+// scalastyle:off method.length
 object AlertEngineMain extends LazyLogging {
   val DAEMON_INTERVAL_MINUTES = 1
-  HttpClientUtil.setConfigurer(new Krb5HttpClientConfigurer())
 
   def main(args: Array[String]): Unit = {
     val parsedArgs = new AlertEngineCliParser(args)
     logger.debug(s"Parsed args: $parsedArgs")
 
     val executorService = Executors.newSingleThreadScheduledExecutor()
 
-    val solrServer = new CloudSolrServer(parsedArgs.zkHost())
+    val solrServer =
```

Comment:
Changed the arguments to take a comma separated list instead of a list of hosts. This will need some good testing